.gitignore

```diff
@@ -1,4 +1,5 @@
 pbnj
+!cmd/pbnj
```

Comment:
this should be `cmd/pbnj/pbnj` right?

Reply:
no

Reply:
oh right, yeah nm. Why not remove the `pbnj` above it though since its no longer relevant and then we can ignore `cmd/pbnj/pbnj`? If you want to keep the other one you can do `/pbnj` so it only applies to the one at the root of the repo.

Reply:
good point, we now build the binary into the `bin/` directory which is already ignored, so this shouldn't be needed anymore.